README.md

```diff
@@ -90,6 +92,55 @@ As an alternative to filters, you can use groupings to limit the amount of artif
 
 The same filter selector keys apply here (except the wildcard character).
 
+### Sort
+
+After all filters are applied it is possible to sort the list of artifacts.
+
+Sort contain two main components: 1) the **key**; and 2) the **order**.
+
+Below is a list of supported keys:
+
+- `set_key`
+- `slot_key`
+- `main_stat_key`
+- `rarity`
+- `level`
+- `rank`
+- `best_score`
+- `refer_id`
+
+And as for the order, the possible parameters are:
+
+- `asc`
+- `desc`
+
+If not specified an order, asc is assumed.
+
+**Special order keys**
+
+By default, sorting use alphabetical character order, but for some keys Genshin's internal order used.
+The purpose of this is to facilitate the mapping between the artifacts exported by G2C and the game.
+
+Below is a list of keys don't use alphabetical order:
+
+- `set_key`
+- `slot_key`
+
+**Genshin order**
+
+This section explains the game order if you want to order the output of artifacts exported by this program in the same way.
+
+- `rarity:desc`
+- `level:desc`
+- `set_key:asc` (specific Genshin order; non-alphabetic order)
+- `slot_key:asc` (specific Genshin order; non-alphabetic order)
+- Original artifact sub stats amount (it would be necessary to calculate the number of bearings based on the known values of each sub stat)
```

Comment:
```suggestion
- Original artifact sub stats amount (it would be necessary to calculate the number of sub stat rolls based on the known values of each sub stat)
```